pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -58,30 +58,9 @@ var (
 	lowPriority, midPriority, highPriority = int32(0), int32(100), int32(1000)
 	mediumPriority                         = (lowPriority + highPriority) / 2
 
-	highPriorityPodInfo = framework.NewPodInfo(&v1.Pod{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      "hpp",
-			Namespace: "ns1",
-			UID:       "hppns1",
-		},
-		Spec: v1.PodSpec{
-			Priority: &highPriority,
-		},
-	})
-	highPriNominatedPodInfo = framework.NewPodInfo(&v1.Pod{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      "hpp",
-			Namespace: "ns1",
-			UID:       "hppns1",
-		},
-		Spec: v1.PodSpec{
-			Priority: &highPriority,
-		},
-		Status: v1.PodStatus{
-			NominatedNodeName: "node1",
-		},
-	})
-	medPriorityPodInfo = framework.NewPodInfo(&v1.Pod{
+	highPriorityPodInfo     = framework.NewPodInfo(st.MakePod().Name("hpp").Namespace("ns1").UID("hppns1").Priority(highPriority).Obj())
+	highPriNominatedPodInfo = framework.NewPodInfo(st.MakePod().Name("hpp").Namespace("ns1").UID("hppns1").Priority(highPriority).NominatedNodeName("node1").Obj())
+	medPriorityPodInfo      = framework.NewPodInfo(&v1.Pod{
 		ObjectMeta: metav1.ObjectMeta{
```

Comment:
Please use PodWrapper as well for medPriorityPodInfo

Reply:
`medPriorityPodInfo` and `unschedulablePodInfo` have annotations which requires #109536 to be merged first since that adds support for Pod annotations so I think that can be a separate PR or I can rebase after.

Reply:
But you've already implemented `CreationTimestamp` in this PR which is included in https://github.com/kubernetes/kubernetes/pull/109536, right?
https://github.com/kubernetes/kubernetes/pull/109536/files#diff-b8c43b6841e30bdec0f6bc6436e7d4c1bb3a67073dfc7efafcc4522429458961R214
So, I think you need to rebase after #109536 merged regardless of this change.

Reply:
Yea that makes sense. I've updated to use PodWrapper for `medPriorityPodInfo` and `unschedulablePodInfo` and will rebase.